src/Athena/Controllers/api/InstitutionCampusesController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Athena.Core.Repositories;
+using Athena.Core.Models;
+using Athena.Exceptions;
+using System.Net;
+
+namespace Athena.Controllers.api
+{
+    public class InstitutionCampusesController : Controller
```

Comment:
Put a route prefix here so you don't have to keep specifying `api/v1/institution/{id}/campuses` in your actions.